src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/extensions */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const first = Big(numberOne);
+  const second = Big(numberTwo);
+  let total = 0;
+  switch (operation) {
+    case '-':
+      total = first.minus(second);
+      break;
+    case 'x':
+      total = first.times(second);
+      break;
+    case '%':
+      total = first / 100;
+      break;
+    case '÷':
+      total = first.div(second);
+      break;
```

Comment:
dont forget to manage division by ZERO on milestone 5...... else it will generate an error;
You've been warned